Atomic/syscontainers.py

```diff
@@ -172,7 +172,7 @@ def install_system_container(self, image, name):
             util.write_out("%s already present" % (name))
             return
 
-        return self._checkout_system_container(repo, name, image, 0, False)
+        return self._checkout_system_container(repo, name, image, 0, False, self.args.remote)
```

Comment:
must be `remote=self.args.remote`, otherwise it gets assigned to `values`.  I have not verified, but this should be enough to fix the failing test.

There is another call to `_checkout_system_container` in the `update_system_container` function that needs the new argument `remote=self.args.remote` as well